tests/ui/components/filters/facetscomponent.js

```diff
@@ -154,3 +160,78 @@ describe('Facets Component', () => {
     expect(remove).toHaveBeenCalledTimes(4);
   });
 });
+
+describe('numerical facets', () => {
+  DOM.setup(document, new DOMParser());
+  setupDOM();
+  const component = setupComponentManager().create('Facets', {
+    container: '#test-component',
+    searchOnChange: true,
+    expand: false
+  });
+  const dynamicFilters = DynamicFilters.fromCore([
+    {
+      fieldId: 'price',
+      displayName: 'Price',
+      options: [
+        {
+          displayName: '0 - 23',
+          count: 20,
+          selected: false,
+          matcher: '$between',
+          value: {
+            start: { matcher: '$ge', value: 0 },
+            end: { matcher: '$lt', value: 23 }
+          }
+        },
+        {
+          displayName: '23 - 60',
+          count: 20,
+          selected: false,
+          matcher: '$between',
+          value: {
+            start: { matcher: '$ge', value: 23 },
+            end: { matcher: '$lt', value: 60 }
+          }
+        }
+      ]
+    }
+  ]);
+  const storage = component.core.storage;
+  storage.set(StorageKeys.DYNAMIC_FILTERS, dynamicFilters);
+  const wrapper = mount(component);
+
+  it('renders numerical facets correctly', () => {
+    const optionLabelNodes = wrapper.find('.js-yxt-FilterOptions-optionLabel--name');
+    expect(optionLabelNodes).toHaveLength(2);
+    expect(optionLabelNodes.at(0).text().trim()).toEqual('0 - 23');
+    expect(optionLabelNodes.at(1).text().trim()).toEqual('23 - 60');
+  });
+
+  it('applies numerical facets correctly', () => {
```

Comment:
Did you consider checking the params passed to core's `verticalSearch`? It might let us test this without needing to be aware of FACET_FILTER_NODES

Reply:
Actually, I think it would be a little messy to try to check the answers-core verticalSearch call, since we would have to use a real instance of the SDK's Core class. Right now we are just using an object with mocked out methods and properties, including `core.verticalSearch` (not `answers-core.verticalSearch`). I updated it to check for `filterRegistry.getFacetsPayload` instead, which ultimately ends up checking the same data, if that's ok?

Reply:
Sounds good